src/Library/demos/Toasts/main.py

```diff
@@ -0,0 +1,54 @@
+import gi
+
+gi.require_version("Gtk", "4.0")
+gi.require_version("GLib", "2.0")
+gi.require_version("Gio", "2.0")
```

Comment:
```suggestion
```

`GLib` and `Gio` don't require specifying the version, only `Gtk` and `Adw` do.

Reply:
Don't think I did, it's only required for libraries that have more than one version (Gtk, GtkSource, etc.) and it's recommended for Adw.

It's also not needed for direct dependencies of Gtk, such as Gdk. 
You can take Workbench itself as a reference for what needs versions: https://github.com/sonnyp/Workbench/blob/main/src/init.js#L1-L7

Reply:
I usually just leave them out until Python complains, and it's only ever complain with Gtk and Adw for me.